DatadogRUM/Sources/RUMVitals/RenderLoop/ViewHitchesReader.swift

```diff
@@ -42,61 +64,84 @@ internal final class ViewHitchesReader: ViewHitchesMetric {
     private var startTimestamp: Double = 0
     private var nextFrameTimestamp: Double?
 
-    private let hangThreshold: TimeInterval
-    private let acceptableLatency: TimeInterval
+    let config: HitchesConfiguration
+    private var _isActive: Bool = false
+    var isActive: Bool { queue.sync { self._isActive } }
 
+    private var hitches: [Hitch] = []
     /// Amount of time when the frames are rendered too late.
-    private var _hitchesDuration: Double = 0.0
-    private var _hitches: [Hitch] = []
-    var hitchesDataModel: HitchesDataModel { queue.sync { (hitches: self._hitches, hitchesDuration: self._hitchesDuration) } }
+    private var hitchesDuration: Double = 0.0
+    var dataModel: HitchesDataModel { queue.sync { (hitches: self.hitches, hitchesDuration: self.hitchesDuration) } }
+
+    private var removedHitchesCount = 0
+    private var ignoredHitchesCount = 0
+    private var startFrameRate: CFTimeInterval?
+    private var didApplyDynamicFraming = false
+    var telemetryModel: HitchesTelemetryModel {
+        queue.sync {
```

Comment:
question/ I'm not sure about the threading model here, do you know what call thread will be waiting for the `sync`?